frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -2005,6 +2029,13 @@ function ReaderFooter:onSuspend()
     end
 end
 
+-- We want to be able to disable auto_refresh_time when displaying *some* non-fullscreen widgets on top of ReaderUI,
+-- otherwise it'll happily keep on ticking and drawing on top of stuff it ought not to... (#6616)
+--- @note: If the widget actually makes it to UIManager's stack (i.e., it's passed to UIManager:show()),
+-- it's generally simpler to set covers_footer when initializing the Widget object.
+ReaderFooter.onDisableFooterAutoRefresh = ReaderFooter.onSuspend
+ReaderFooter.onRestoreFooterAutoRefresh = ReaderFooter.onResume
```

Comment:
Looks like you're no more using these ?

Reply:
Currently, no, but they may come in handy if we need to do this for subwidgets that never make it to the stack (i.e., never passed to `UIManager:show`).

Which, granted, will potentially never happen ^^.

I guess I could leave it, but commented out, then?

Reply:
Yes, I'd rather see them commented out, so I won't wonder why/how these are used if I ever look at that footer again :)